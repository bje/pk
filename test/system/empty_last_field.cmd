../../src/fieldx -d: -e 1 5 7
