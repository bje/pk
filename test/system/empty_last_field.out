A E NULL
a e NULL
